CMakeLists.txt

```diff
@@ -3,6 +3,13 @@ list( APPEND CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake )
 
 project( CroftEngine VERSION 1.9.1 )
 
+if (WIN32) 
+    set(CMAKE_CONFIGURATION_TYPES Debug RelWithDebInfo CACHE TYPE INTERNAL FORCE )
```

Comment:
What's this line supposed to do?

Reply:
It prevents building Release with a multi-configuration build system like Visual Studio.